src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -1541,8 +1541,24 @@ public function deleteProperties(array $operations): void
     {
         $this->assertLoggedIn();
 
+        // Building a table describing which properties need to be deleted from which nodes so that we only have to parse contents once
```

Comment:
```suggestion
        // Building a table describing which properties need to be deleted from which nodes so that we only have to parse contents of each node once
```